bench-tps/src/bench_tps_client/thin_client.rs

```diff
@@ -1,6 +1,7 @@
 use {
```

Comment:
Plan is still for ThinClient to be deprecated then deleted, right?

Reply:
Yes, been deprecated in [PR: #35365](https://github.com/solana-labs/solana/pull/35365). Fully removing it from other parts is WIP